src/API.ts

```diff
@@ -178,7 +178,7 @@ function save(
     'stream' in files ? (files as unknown as PenumbraFile) : files[0];
   const [
     filename,
-    extension = file.mimetype ? mime.extension(file.mimetype) : '',
+    extension = file.mimetype ? `.${getExtension(file.mimetype)}` : '',
```

Comment:
lighter mime type. also noticed neither extension returns with a `.` prefix which would result in `imagepng`